test/controller_test.exs

```diff
@@ -0,0 +1,55 @@
+defmodule ControllerTest do
+  use ExUnit.Case
+  import Controller
+  import ExUnit.CaptureIO
+
+  describe "parse_input/1" do
+    test "when passed a character on the menu, does not error" do
+      output = fn -> parse_input("V") end
+      assert capture_io(output) == "Which recipe would you like to view?\n- Ice Cubes\n\n"
```

Comment:
So these tests are fine if our interface is always the command line, right? Can you think of a way to make these more robust to whatever interface we choose to use?

Reply:
i.e. how could we write these tests without the use of `capture_io`?